app/executor.php

```diff
@@ -322,8 +322,8 @@ function getStorageDevice($root): Device
             $endTime = \time();
             $container = array_merge($container, [
                 'status' => 'ready',
-                'response' => \mb_strcut($stdout, 0, 1000000), // Limit to 1MB
-                'stderr' => \mb_strcut($stderr, 0, 1000000), // Limit to 1MB
+                'response' => \mb_strcut($stdout, 0, App::getEnv('_APP_FUNCTIONS_RESPONSE_SIZE_LIMIT', 1000000)), // Default Limit to 1MB
+                'stderr' => \mb_strcut($stderr, 0, App::getEnv('_APP_FUNCTIONS_RESPONSE_SIZE_LIMIT', 1000000)), // Default Limit to 1MB
```

Comment:
Thanks for making this PR. I'm not sure if only changing the limit is the best approach to will really solve the problem since the data is also stored in the database which has a finite length as well:

https://github.com/appwrite/appwrite/blob/648b6f2038fb59b2beb78d81cea08aa730a106c3/app/config/collections.php#L2367-L2388